helpers/calendar/analytics_helper.py

```diff
@@ -104,38 +104,37 @@ def get_all_events_in_a_room(self, calendar_id, min_limit, max_limit):
             - calendar_id - for specific room
             - min_limit, max_limit(Time range)
         """
-        try:
-            events_result = get_google_calendar_events(
-                calendarId=calendar_id, timeMin=min_limit,
-                timeMax=max_limit, singleEvents=True,
-                orderBy='startTime')
-        except Exception:
-            raise GraphQLError("Resource not found")
-        all_events = events_result.get('items', [])
-        return all_events
+        events = EventsModel.query.all()
+        result = []
+        for event in events:
+            event = {
+                'room_id': event.room_id,
+                'min_limit': event.start_time,
```

Comment:
@nakatuddesuzan I suggest that these keys `min_limit` and `max_limit` be renamed to something like `lower_date_limit` and `upper_date_limit` just to make them a bit more descriptive.